content/docs/2.8/scalers/aws-sqs.md

```diff
@@ -28,8 +28,9 @@ triggers:
 - `queueURL` - Full URL for the SQS Queue. The simple name of the queue can be used in case there's no ambiguity.
 - `queueLength` - Target value for queue length passed to the scaler. Example: if one pod can handle 10 messages, set the queue length target to 10. If the actual messages in the SQS Queue is 30, the scaler scales to 3 pods. (default: 5)
 
-> For the purposes of scaling, "actual messages" is equal to `ApproximateNumberOfMessages` + `ApproximateNumberOfMessagesNotVislble`, since `NotVisible` in SQS terms means the message is still in-flight/processing.
+> For the purposes of scaling, the default formula for "actual messages" is equal to `ApproximateNumberOfMessages` + `ApproximateNumberOfMessagesNotVislble`, since `NotVisible` in SQS terms means the message is still in-flight/processing. If you wish to only scale on `ApproximateNumberOfMessages` set `scaleOnInFlight` to `false`.
 
+- `scaleOnInFlight` - Bool flag to set the metrics to scale on. When set to `false` "actual messages" is equal to `ApproximateNumberOfMessages`. When set to `true` "actual messages" is equal to `ApproximateNumberOfMessages` + `ApproximateNumberOfMessagesNotVislble`, since `NotVisible` in SQS terms means the message is still in-flight/processing. (default: true)
```

Comment:
Just a small suggestion
```suggestion
- `scaleOnInFlight` - Indication whether or not to scale on queued messages or to include in-flight messages as well.
  - When set to `false` "actual messages" is equal to `ApproximateNumberOfMessages`. When set to `true` "actual messages" is equal to `ApproximateNumberOfMessages` + `ApproximateNumberOfMessagesNotVislble`, since `NotVisible` in SQS terms means the message is still in-flight/processing. (default: true)
```

Reply:
LGTM , I like this wording more. I'll make a the change

Reply:
@LiamStorkey could you please update this PR so we can merge the feature?